studiopress/atomic-blocks
# Rebrand Getting Started page

**Summary of change:**
Change styling of Getting Started page if ABP/GPB is found.

- Add new GPB branding and colors
- Customize content specific to GPB
- Remove FontAwesome dependency and replace with inline SVG 

**How to test:**
Make sure you have ABP/GPB installed and visit the Getting Started page. You can reverse the `atomic_blocks_is_pro` flag to check the AB version of the Getting Started page.

**Suggested Changelog Entry:**
Improve Getting Started page styling and performance.
